app/lib/common/gatewayMethods.js

```diff
@@ -444,7 +447,7 @@ export function validateAddress({
 }
 
 let _conversionCache = {};
-export function getConversionJson(inputs) {
+export function getConversionJson(inputs, isUserAuthorized = false) {
```

Comment:
Preferrably would be to have the custom code within the a blocktrades module, independent of actual storage solutions.

i.e. can we have the method signature be` getConversionJson(inputs, userAccessToken = null)`, and if null old logic?

Reply:
Yes, it is good to use:
```

Reply:
Will you be refactoring this?

Reply:
@sschiessl-bcp yes, I can add next commit.